burotel/indico_burotel/plugin.py

```diff
@@ -8,19 +8,34 @@
 import os
 from datetime import datetime, time
 
-from flask import current_app, redirect, request, url_for
+from flask import after_this_request, current_app, redirect, request, url_for
+from flask_babelex import format_date
```

Comment:
We no longer have flask_babelex - should probably use the one from flask_babel or the one from the indico core